packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -9,11 +9,11 @@
     flex-direction: column;
     position: fixed;
     // This will ensure that by default, the notification dialog sizes to the viewport (minus the insets).
-    width: calc(100% - 2 * var(--terra-notification-dialog-small-horizontal-inset, 10px));
+    width: calc(100% - 2 * var(--terra-notification-dialog-small-horizontal-inset, 17px));
 
     // This will ensure that above the medium breakpoint, a max-width is enforced
```

Comment:
This comment now needs to be updated?

Reply:
Updated here 9fe343e5697fe8f6c458f5b8d49a28635e5fa77c